ansys/dpf/core/examples/downloads.py

```diff
@@ -728,3 +728,121 @@ def download_piston_rod(should_upload: bool = True, server=None, return_local_pa
 
     """
     return _download_file("piston_rod", "piston_rod.rst", should_upload, server, return_local_path)
+
+
+def download_d3plot_beam(should_upload: bool = True, server=None, return_local_path=False) -> list:
+    """Download the result file of an example of a d3plot file with beam elements
+    and return the download path available server side.
+    If the server is remote (or doesn't share the memory), the file is uploaded or made available
+    server side.
+
+    Examples files are downloaded to a persistent cache to avoid
+    re-downloading the same file twice.
+
+    Parameters
+    ----------
+    should_upload : bool, optional (default True)
+        Whether the file should be uploaded server side when the server is remote.
+    server : server.DPFServer, optional
+        Server with channel connected to the remote or local instance. When
```

Comment:
A channel (singular) or channels (plural)?

Reply:
Singular, it should be kept as is, I think. Thanks